tests/Mysql/InsertTest.php

```diff
@@ -58,6 +58,21 @@ public function testHighPriority()
         $this->assertSameSql($expect, $actual);
     }
 
+    public function testOrReplace()
+    {
+        $this->query->orReplace()
+                    ->into('t1')
+                    ->cols(array('c1', 'c2', 'c3'))
+                    ->set('c4', 'NOW()')
+                    ->set('c5', null);
+
+        $actual = $this->query->__toString();
+        $expect = sprintf($this->expected_sql_with_flag, '');
+        $expect = preg_replace('|INSERT |ms', 'REPLACE', $expect);
+
```

Comment:
It would have been good if you could write the raw query itself than using a preg_replace. What I mean is something as https://github.com/auraphp/Aura.SqlQuery/pull/115/files#diff-f685026c17cbf1227a63eb3b344b96a4R27 .